airflow/dev/docker-compose-data-ingestion.yml

```diff
@@ -15,6 +21,12 @@ services:
             - webserver
         environment:
             - PORT=8080
+            # Ensure that the project set with PROJECT_ID has a BigQuery dataset named 'data_sources'
+            - DATASET_NAME=data_sources
```

Comment:
This name isn't compatible with the TEST environment, which is called 'test_data_sources'

Reply:
Whoops. sorry I forgot that I actually named it somewhere. The schema from the above comment is fine. We should just have the names here and in terraform align.

Reply:
Good catch. I'll use the dataset name we defined in terraform. 

FYI: From the thread above on the BQ structure, we'll probably need to refactor this and the terraform file.